frontend/src/components/PlantsView/SortBy.tsx

```diff
@@ -0,0 +1,19 @@
+import {PlantSummary} from "../../api";
```

Comment:
Dobry pomysł z przeniesieniem tych typów i funkcji poza komponent, ale wrzuciłbym to wszystko do 1 pliku `utils.ts` w tym samym folderze (tak robiliśmy np. w komponencie kalendarza).

Rozszerzenie tsx używamy w plikach reactowych, te funkcje to czysty typescript

Reply:
Przeniesione